x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/waterfall_helpers/waterfall_helpers.ts

```diff
@@ -283,6 +289,7 @@ export function getWaterfall(
     items,
     itemsById,
     getTransactionById,
+    errorsPerTransaction,
```

Comment:
nit: The naming seem inconsistent between `errorsPerTransaction` and `itemsById`. They are both dictionaries with id as key, right? 
If changing it causes more harm than good, just leave it.

Reply:
changed to 'errorCountByTransactionId' to be consistent with the other lookup object